DDECal/DDECal.cc

```diff
@@ -978,14 +970,15 @@ bool DDECal::process(const DPBuffer& bufin) {
   return false;
 }
 
-void DDECal::idgCallback(size_t row, size_t direction, size_t dataDescId,
+void DDECal::idgCallback(size_t row, size_t direction,
```

Comment:
Did you remove this because we only support one datadescid (which is typically a spectral window)? In the future we may want to support more, maybe leave it in here and add an assert that it's zero.  That'll help future programmers (me).

Reply:
That's indeed the reason why I removed it. I'll store the FacetPredict::PredictCallback, so it includes the dataDescId again, and I'll also update the lambda and idgCallback.